xc/device.ml

```diff
@@ -1404,6 +1404,8 @@ module Vusb = struct
     | _ ->()
 
   let vusb_plug ~xs ~domid ~id ~hostbus ~hostport ~version =
+    if not (is_upstream_qemu domid) then
```

Comment:
This is a low-level function. I think this should use the `Profile` module:

```
let device_model = Profile.of_domid domid in
if device_model <> Qemu_upstream) then ..
```

The error message should also use the profile terminology:
```
"Failed to plug VUSB %s because domain %d uses device-model profile  %s" 
  (...) domid (Profile.string_of device_model)
```

Reply:
updated.